honeybee_radiance/cli/translate.py

```diff
@@ -0,0 +1,197 @@
+
+        # translate the model to a radiance folder
+        log_file.write('Translating Model to Radiance folder.\n')
+        rad_fold = model.to.rad_folder(model, folder, folder_type, minimal)
+        log_file.write('Model translation successful.\n')
+        log_file.write('Radiance folder output to: {}'.format(rad_fold))
+    except Exception as e:
+        _logger.exception('Model translation failed.\n{}'.format(e))
+        sys.exit(1)
+    else:
+        sys.exit(0)
+
+
+@translate.command('model-to-rad')
+@click.argument('model-json')
+@click.option('--blk', help='Boolean to note whether the "blacked out" version '
+              'of the geometry should be output, which is useful for direct studies '
+              'and isolation studies of individual apertures.',
+              default=False, show_default=True)
+@click.option('--minimal', help='Boolean to note whether the radiance strings should '
+              'be written in a minimal format (with spaces instead of line breaks).',
+              default=False, show_default=True)
+@click.option('--log-file', help='Optional RAD file to output the RAD string of the '
+              'translation. By default this will be printed out to stdout',
+              type=click.File('w'), default='-')
+def model_to_rad(model_json, blk, minimal, log_file):
+    """Translate a Model JSON file to a Radiance string.
+    \n
+    The resulting strings will include all geometry (Rooms, Faces, Shades, Apertures,
+    Doors) and all modifiers. However, it does not include any states for dynamic
+    geometry and will only write the default state for each dynamic object. To
+    correctly account for dynamic objects, model-to-rad-folder should be used.
+    \n
+    Args:
+        model_json: Full path to a Model JSON file.
+    """
+    try:
+        # check that the model JSON is there
+        assert os.path.isfile(model_json), \
+            'No Model JSON file found at {}.'.format(model_json)
+
+        # re-serialize the Model to Python
+        with open(model_json) as json_file:
+            data = json.load(json_file)
+        model = Model.from_dict(data)
+
+        # translate the model to a rad string and put it in a dictionary
+        model_str, modifier_str = model.to.rad(model, blk, minimal)
+        rad_dict = {'model_str': model_str, 'modifier_str': modifier_str}
+
+        # write out the rad string
+        log_file.write(json.dumps(rad_dict))
+    except Exception as e:
+        _logger.exception('Model translation failed.\n{}'.format(e))
+        sys.exit(1)
+    else:
+        sys.exit(0)
+
+@translate.command('modifiers-to-rad')
+@click.argument('modifier-json')
+@click.option('--minimal', help='Boolean to note whether the radiance strings should '
+              'be written in a minimal format (with spaces instead of line breaks).',
+              default=False, show_default=True)
+@click.option('--log-file', help='Optional RAD file to output the RAD string of the '
```

Comment:
I think naming them log-file can be confusing. They are really output files. I have used log-file for a command like separate sensor grid because the pts files is the real output and log-file logs the info about the grids. This is a generic comment about all the commands.

Reply:
So are you saying that we should change all of the '--log-file' options across all of the CLI commands to be called `--output-file` instead?  If so, we can open an issue for it and I'll make the change when I can.

Reply:
Yes. and since this is a breaking change I think we should do it sooner than later.

Reply:
https://github.com/ladybug-tools/honeybee-radiance/issues/121

Reply:
Understood. I will open similar issues on the honeybee-energy and dragonfly repos.